### ISSUE TITLE
Wrong package version

### ISSUE THREAD

**Issue by NeonWizard** [bndr/pipreqs#223]
Using pipreqs gives me `discord.py==1.3.4`, where `pip3 freeze` gives me `discord.py==1.5.1`. I don't have any other versions of Python installed and `--use-local` gives me `1.3.4` as well.

**Comment by pkalemba** [comment]
can you provide some logs from app? please run it in verbose mode and paste logs :)

**Comment by NeonWizard** [comment]
```
> pipreqs --print --debug
DEBUG: Found packages: {'asyncio', 'time', 'os', 'sys', 'datetime', 'random', 'discord'}
DEBUG: Found imports: discord
DEBUG: Getting packages information from Local/PyPI
DEBUG: Starting new HTTPS connection (1): pypi.python.org:443
DEBUG: https://pypi.python.org:443 "GET /pypi/discord/json HTTP/1.1" 301 122
DEBUG: Starting new HTTPS connection (1): pypi.org:443
DEBUG: https://pypi.org:443 "GET /pypi/discord/json HTTP/1.1" 200 1255
DEBUG: Writing 2 requirements: discord.py, discord to -
discord.py==1.3.4
discord==1.0.1
INFO: Successfully output requirements

> pip3 freeze | grep discord
discord.py==1.5.1
```

**Comment by pkalemba** [comment]
I opened https://github.com/bndr/pipreqs/pull/224 which should fix this

**Comment by alan-barzilay** [comment]
I believe this was closed on PR #224, feel free to reopen it if the issue is not fixed

**Comment by pkalemba** [comment]
I have no power to close it 😃
